src/components/body/body-cell.component.ts

```diff
@@ -34,7 +76,11 @@ import { MouseEvent, KeyboardEvent } from '../../events';
         [ngTemplateOutletContext]="cellContext">
       </ng-template>
     </div>
-  `
+  `,
+  styles: [
```

Comment:
These should be in the main css file.

Reply:
Can you point me to the CSS file? I am unable to find it.

Reply:
I think he's talking about src/components/datatable.component.scss